sites/platform/src/dedicated-environments/overview.md

```diff
@@ -0,0 +1,69 @@
+
+![The grid architecture](/images/dedicated/grid-architecture.svg "0.50")
+
+## Deployment
+
+With a Dedicated environment, you are given the freedom to deploy into **any region of supported IaaS providers** (currently **AWS, Azure, GCP, OVH**). This differs from the Grid architecture, which is solely available in [public regions](https://platform.sh/regions/). 
+
+For a full  list of public regions and IP addresses, visit the [Regions page](https://docs.platform.sh/development/regions.html#regions).
+
+In a Grid region, incoming and outgoing traffic is handled via central region gateways, and [publicly available IP addresses](https://docs.platform.sh/development/regions.html#public-ip-addresses) can be used for external firewalls. The public IP addresses for these public regions are stable but not guaranteed never to change.
+
+## **Grid vs Dedicated** 
+
+Whether you choose a Grid or Dedicated environment depends on the needs you have. In the table below, you can see the different ways in which either environment might work for you: 
+
+| FEATURE | GRID | DEDICATED |
+| --- | --- | --- |
+| **SLA** | 99.9% | 99.99% |
+| **Infrastructure** | Containers with dedicated resources on top of a shared redundant infrastructure | Dedicated 3 node clusters |
+| **Functioning** | A single container is deployed per runtimes and per services | Three nodes are running all applications and services are replicated across all of them |
+| **Resources allocation** | Resources are spread through one container with fixed sizes after deployment | Resources are deployed across 3 nodes |
+| **Usable regions** | Only the publicly available regions can be used | Any region can be used to deploy |
+| **Autonomous upsize** | Yes | Managed through Platform.sh |
+| **Upsize and downsize methods** | Redeploy - possible downtime depending on the hooks | No downtime - each instance is upsize in a rolling fashion |
+| **Multi-app support** | Supported natively | Supported through docroots with additional billing |
+| **Production branch** | Master or main branch | Dedicated production branch |
+| **Custom domains name** | Supported on master and any other branches | Supported on the production environment |
+| **Sync and merge functionalities** | Yes on all branches | Only on Development environments |
+| **Environment clone** | Yes on all branches | Only on Development environments |
+| **MySQL Replication** | None: standalone service container | Yes: 3 services nodes cluster |
+| **Redis Replication** | None: standalone service container | Yes: 3 services nodes cluster |
+| **MongoDB** | Standalone service container | Yes |
```

Comment:
we have "yes" under Dedicated but in the DG2 overview for MongoDB we have "Not supported" and it's not listed at all on the DG3 overview.

Reply:
Just clarified with the mega spreadsheet I was working from. It had "in progress" in some places and "not supported in others". I've put it down as a no in all cases as now :)